apps/omg_api/lib/root_chain_coordinator/core.ex

```diff
@@ -89,8 +89,8 @@ defmodule OMG.API.RootChainCoordinator.Core do
     service.synced_height <= service_reported_sync_height and state.root_chain_height >= service_reported_sync_height
   end
 
-  defp get_services_to_sync(state, previous_synced_height) do
-    case get_synced_height(state) do
+  defp get_services_to_sync(state, service_name, previous_synced_height) do
+    case get_synced_height(state, service_name) do
```

Comment:
here you send service_name? 
it should be pid

Reply:
Nope, it should be `service_name`